WcaOnRails/config/locales/locales.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+﻿# frozen_string_literal: true
```

Comment:
This is the relevant part of the travis failure: https://travis-ci.org/thewca/worldcubeassociation.org/builds/196437173#L457-L460